ClientApp/viking-vault/src/ProfilePicture.tsx

```diff
@@ -1,14 +1,57 @@
 import * as React from "react";
 import profilePicture from "./UI/user2.png";
 import './ProfilePicture.css';
+import { constants } from "./ConstantVariables";
 
-class ProfilePicture extends React.Component<any, any> 
+const API_URL = `${constants.baseUrl}userprofilepages`;
+
+interface IProfilePicture{
+    userProfilePicture: string
+}
+
+class ProfilePicture extends React.Component<any, IProfilePicture> 
 {
+    constructor(props: any){
```

Comment:
You don't need a constructor if all you do is set up the state, simply use
` state = { userProfilePicture: profilePicture } `
in the class, same way as you'd define a class property